makefile

```diff
@@ -61,7 +61,7 @@ TESTSRCS := $(shell find $(TESTDIR) -name '*.cpp')
 TESTOBJS := $(patsubst $(TESTDIR)/%.cpp,$(TESTOBJDIR)/%.o,$(TESTSRCS))
 TESTFOLDERS := $(sort $(dir $(TESTSRCS)))
 TESTLDFLAGS := -L./
-TESTLIBS := -lgtest -lgtest_main -lpthread -lOP2Utility
+TESTLIBS := -lgtest -lgtest_main -lpthread -lOP2Utility -lstdc++fs
```

Comment:
I had not thought about this being a problem. Thank you for catching and fixing.